x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -1,5 +1,71 @@
 {
   "properties": {
+    "alerts": {
+      "properties": {
+        "count_total": {
+          "type": "long"
+        },
+        "count_active_total": {
+          "type": "long"
+        },
+        "count_disabled_total": {
+          "type": "long"
+        },
+        "throttle_time": {
+          "properties": {
+            "min": {
+              "type": "long"
+            },
+            "avg": {
+              "type": "float"
+            },
+            "max": {
+              "type": "long"
+            }
+          }
+        },
+        "schedule_time": {
+          "properties": {
+            "min": {
+              "type": "long"
+            },
+            "avg": {
+              "type": "float"
+            },
+            "max": {
+              "type": "long"
+            }
+          }
+        },
+        "connectors_per_alert": {
+          "properties": {
+            "min": {
+              "type": "long"
+            },
+            "avg": {
+              "type": "float"
+            },
+            "max": {
+              "type": "long"
+            }
+          }
+        },
+        "count_active_by_type": {
+          "properties": {
+            "DYNAMIC_KEY": {
```

Comment:
Same as comment https://github.com/elastic/kibana/pull/78832#discussion_r497796848
Having a placeholder for unknowns is a great way to keep track of fields that are probably unmapped right now. If Infra isn't comfortable with filtering out the `DYNAMIC_KEY` fields, we'll have to take care of that in the diff we provide at FF (i.e. ignore these fields because they can't be mapped in the legacy service indices anyway).